ietf/api/urls.py

```diff
@@ -56,6 +57,9 @@
     url(r'^version/?$', api_views.version),
     # Application authentication API key
     url(r'^appauth/[authortools|bibxml]', api_views.app_auth),
+    # latest versions
+    url(r'^rfcdiff-latest-json/%(name)s(?:-%(rev)s)?(\.txt|\.html)?/?$' % settings.URL_REGEXPS, api_views.rfcdiff_latest_json),
```

Comment:
Sorry to ask for multiple changes, but it occurs to me that this would be better as a permanent redirect in case there are other users of the endpoint we're not yet aware of.

Reply:
No worries, I have updated the `ietf/doc/urls.py` URLs as permanent redirects.